ipalib/install/certmonger.py

```diff
@@ -670,11 +670,3 @@ def wait_for_request(request_id, timeout=120):
         raise RuntimeError("request timed out")
 
     return state
-
```

Comment:
Why did you remove the test block?

Reply:
Because it was broken and generally unused AFAIK so I didn't bother to fix it.

I can fix it if desired, I found it useful during development, but it dated back to the very first commit virtually unchanged (2010).